src/app/components/PriceChart.tsx

```diff
@@ -316,3 +354,126 @@ export function PriceChart() {
     </>
   );
 }
+
+export function PairInfoTab() {
+  const t = useTranslations();
+  const { pairsList } = useAppSelector((state) => state.rewardSlice);
```

Comment:
We have the `pairsList` available in both the RewardSlice and also pairSelector. I suggest to use the one from pairSelector instead:
```suggestion
  const { pairsList } = useAppSelector((state) => state.pairSelector);
```